Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SB-1-B.yml

```diff
@@ -28,12 +28,8 @@ shortDescription:
 
 #A multi-paragraph description of the activity
 longDescription: |
-  Keep a record of all dependencies used throughout the target production environment. This is sometimes referred to as a Bill of Materials (BOM). Consider that the different dependencies and aspects of the application may consume entirely different dependencies. For example, if the software package is a web application, cover both the server-side application code and client-side scripts. In building these records, consider the various locations where dependencies might be specified:
-
-  * configuration files 
-  * the project's directory on disk
-  * package management tool
-  * code (e.g. via an IDE that supports listing dependencies)
+  Keep a record of all dependencies used throughout the target production environment. This is sometimes referred to as a Bill of Materials (BOM). Consider that different components of the application may consume entirely different dependencies. For example, if the software package is a web application, cover both the server-side application code and client-side scripts. In building these records, consider the various locations where dependencies might be specified such as 
+ configuration files, the project's directory on disk, a package management tool or the actual code (e.g. via an IDE that supports listing dependencies).
```

Comment:
I think the paragraph shouldn't be divided into two.

Reply:
Done